src/Umbraco.Web.Common/Security/MembersUserManager.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Principal;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Umbraco.Core;
+using Umbraco.Core.Configuration.Models;
+using Umbraco.Core.Models.Membership;
+using Umbraco.Core.Security;
+using Umbraco.Extensions;
+using Umbraco.Infrastructure.Security;
+using Umbraco.Net;
+using Umbraco.Web.Models.ContentEditing;
+
+
+namespace Umbraco.Web.Common.Security
+{
+    public class MembersUserManager : UmbracoUserManager<MembersIdentityUser, MemberPasswordConfigurationSettings>, IMembersUserManager
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public MembersUserManager(
+            IIpResolver ipResolver,
+            IUserStore<MembersIdentityUser> store,
+            IOptions<MembersIdentityOptions> optionsAccessor,
+            IPasswordHasher<MembersIdentityUser> passwordHasher,
+            IEnumerable<IUserValidator<MembersIdentityUser>> userValidators,
+            IEnumerable<IPasswordValidator<MembersIdentityUser>> passwordValidators,
+            //TODO: do we need members versions of this?
+            BackOfficeLookupNormalizer keyNormalizer,
```

Comment:
Nope just needs to be named more generic. We also have 2x test cases for the same exact thing: NopLookupNormalizerTests + BackOfficeLookupNormalizerTests both are just testing BackOfficeLookupNormalizer :P 

Don't really think we need to pass this in with DI and instead just pass a new instance of this in the base class.

Reply:
Gotcha. Will merge the duplicate tests and rename to generic - `NoOpLookupNormalizer` perhaps?

Then I'll look into the shift away from DI

Reply:
As an aside there is a TODO on the normalizer that says "Do we need this" - I'd like to delete the TODO, what do we need to know to know if we need it?

Reply:
Another side note, `BackOfficeIdentityErrorDescriber` is empty at the moment. Do I need to create an also empty `MemberIdentityErrorDescriber` or should I leave it for now?